src/include/isns/dialogs.rb

```diff
@@ -166,7 +165,19 @@ def initialize_isns_dialogs(include_target)
     #
     # @return [::CWM::ServiceWidget]
     def service_widget
-      @service ||= ::CWM::ServiceWidget.new(IsnsServer.service)
+      @service_widget ||= ::CWM::ServiceWidget.new(IsnsServer.service)
+    end
+
+    # Add the service wiget if is not already included
+    #
+    # Kind of lazy initialization, since the "open-isns" must be installed in the system.
+    # Otherwise it crashes
+    #
+    # @see {Yast::IsnsComplexInclude.ReadDialog}
```

Comment:
NP: Do not use braces here, simply  `# @see Yast::IsnsComplexInclude.ReadDialog`.